src/Microsoft.DotNet.Tools.Builder/README.md

```diff
@@ -0,0 +1,32 @@
+dotnet-build
+===========
+
+**NAME** 
+dotnet-build -- Orchestrates the compilation of a project and all its dependencies.
+
+**SYNOPSIS**
+dotnet build[options]
+
+**DESCRIPTION**
+
+The build verb orchestrates the compilation of a project: it gathers the dependencies of a project and decides which to compile. 
+
+Users should invoke the Build verb when they want the entire dependency graph compiled, and Compile when they want only a specific project compiled.
```

Comment:
Issue #532 shows the proposed user experience with the Build verb and explains the difference in responsibilities.

Scenarios for using each verb:

The user wants to build a project and all its dependencies. If the incremental safety checks are passed, then the build process is incremental as well. 
The user types: `dotnet build path/to/project`

The user wants to compile only a specific project, without its dependencies, and without incrementality. The user types 'dotnet compile path/to/project'
If the user wants to compile the entire dependency graph with the Compile verb, he needs to call Compile on each dependency.

In summary, the Build verb takes over the Compile verb and users should use Build instead of Compile (unless they want to target specific projects with Compile).